docs/www/quick-start-kubernetes.md

```diff
@@ -4,91 +4,149 @@ order: 0
+    Then, create a cluster with:
 
-```
-eksctl create cluster \
---name redpanda \
---nodegroup-name standard-workers \
---node-type m5.xlarge \
---nodes 3 \
---nodes-min 1 \
---nodes-max 4 \
---node-ami auto
-```
+    ```
+    gcloud container clusters create redpanda --machine-type n1-standard-4
+    ```
 
-This command will take 10-15 minutes to complete. Once it is completed then
-follow the [Helm install instructions](#Using-Helm-to-Install-Redpanda).
+  </tab>
+</tabs>
 
-## Google GKE
-
-First complete the "Before You Begin" steps describe in
-[Google Kubernetes Engine Quickstart](https://cloud.google.com/kubernetes-engine/docs/quickstart).
-Once complete you can create a cluster using the following command:
-
-```
-gcloud container clusters create redpanda --machine-type n1-standard-4
-```
+## Using Helm to Install Redpanda
 
-Once the command completes then follow the
-[Helm install instructions](#Using-Helm-to-Install-Redpanda)
\ No newline at end of file
+1. Using Helm, add the Redpanda chart repository and update it:
+
+    ```
+    helm repo add redpanda https://charts.vectorized.io/ && helm repo update
+    ```
+
+2. Install the Redpanda operator CRDs:
+
+    ```
+    kubectl apply -k 'https://github.com/vectorizedio/redpanda/src/go/k8s/config/crd?ref=<latest version>'
+    ```
+
+    You can find the latest version number of the operator on the [list of operator releases]().
+
+3. Install Redpanda on your Kubernetes cluster with:
+
+    ```
+    helm install --namespace redpanda-system --create-namespace redpanda-operator redpanda/redpanda-operator
+    ```
+
+## Next steps
+
+After you set up Redpanda in your Kubernetes cluster, you can:
+
+- Install resources from our sample files:
+    - [One node cluster](kubectl apply -f https://raw.githubusercontent.com/vectorizedio/redpanda/dev/src/go/k8s/config/samples/one_node_cluster.yaml)
+    - [Cluster with external connectivity (verified on AWS)](kubectl apply -f https://raw.githubusercontent.com/vectorizedio/redpanda/dev/src/go/k8s/config/samples/external_connectivity.yaml)
+    - [Cluster with TLS encryption](kubectl apply -f https://raw.githubusercontent.com/vectorizedio/redpanda/dev/src/go/k8s/config/samples/redpanda_v1alpha1_with_tls.yaml) 
+        To use this cluster you have to mount TLS configuration for rpk.
+        The following definition creates a pod that has TLS configured.
+        From that pod, users can list, create, and delete topics, and also produce and consume events.
+
+        ```
+        cat <<EOF | kubectl apply -f -
+        apiVersion: v1
+        kind: ConfigMap
+        metadata:
+         name: rpk-config
+        data:
+         redpanda.yaml: |
+           redpanda:
+           rpk:
+             tls:
+               truststore_file: /etc/tls/certs/ca.crt
+        ---
+        apiVersion: v1
+        kind: Pod
+        metadata:
+         name: produce-msg
```

Comment:
ntw. this has to run in the same namespace as you used to install redpanda cluster

Reply:
Where do you specify the namespace for installation?

Reply:
I added `-n redpanda-system` to the commands. Is that right?

Reply:
```suggestion
         name: produce-msg
         namespace: test-namespace
```


or any other namespace name